Misc/NEWS.d/next/Tests/2021-09-25-11-05-31.bpo-45280.3MA6lC.rst

```diff
@@ -0,0 +1 @@
+Adds a test case for empty :class:`typing.NamedTuple`.
```

Comment:
Here and in the commit message I would use “Add”, not “Adds”. This conforms better to our style.

Reply:
Thanks! Updated.